lib/imageRouter/filesystem.js

```diff
@@ -47,7 +29,7 @@ exports.uploadImage = function (imagePath, callback) {
 
   let filename = path.basename(imagePath)
   try {
-    filename = pickFilename(path.basename(imagePath))
+    filename = randomFilename()
```

Comment:
the filename will no longer with file extension (.png), we should modify pickFilename function and only use randomFilename their without default filename.